packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -262,7 +262,10 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
             // sign workflow in Actions:
             // signTransaction > sign[COIN]Transaction > requestPushTransaction (modal with promise decision)
```

Comment:
Also update comment here (also have a look for other occurences of changed naming withi at least the files that will be in send form package)

Reply:
fixed by 177cf1b5cc